src/store/modules/user/actions.ts

```diff
@@ -52,22 +52,37 @@ const actions: ActionTree<UserState, RootState> = {
   /**
    * Get User profile
    */
-  async getProfile ( { commit }) {
+  async getProfile ( { commit, dispatch }) {
     const resp = await UserService.getProfile()
     if (resp.status === 200) {
       const localTimeZone = moment.tz.guess();
       if (resp.data.userTimeZone !== localTimeZone) {
         emitter.emit('timeZoneDifferent', { profileTimeZone: resp.data.userTimeZone, localTimeZone});
       }
+
       commit(types.USER_INFO_UPDATED, resp.data);
+      await dispatch('getEComStores', { facilityId: resp.data.facilities[0] })
```

Comment:
Check for facilities in resp before dispatching getEComStores.

Reply:
Updated ma'am